backend-api/app/Http/Controllers/ProfileController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Http\Controllers;
+use App\User;
+use Illuminate\Http\Request;
+
+
+class ProfileController extends Controller
+{
+    public function show($id)
+    {
+        $users = get::table('users')->where('id',$id)->get();
+        return view('user.profile',compact('users'));
```

Comment:
Do not return a view! Why you have a laravel view here? It should be a REST `response()` and use it at React

Reply:
We got rid of that controller. We are not using it.